printer/table/printer.go

```diff
@@ -33,6 +34,20 @@ func entryFromStruct(obj reflect.Value, h []string) []string {
 			values = append(values, value.String())
 		case reflect.Uint32:
 			values = append(values, strconv.FormatInt(int64(value.Uint()), 10))
+		case reflect.Slice:
+			// Render Slice of strings
+			typ := value.Type().Elem()
+			if typ == reflect.TypeOf(string("")) {
```

Comment:
Is it necessary ? We already know that `value` is a slice, we can directly iterate on it. :thinking:

Reply:
We can but we don't know if it's a slice of string, int or whatever right ?

Reply:
This is what we're doing later: 
https://github.com/cycloidio/cycloid-cli/blob/add905af0aa41c680a6180a8bfa521434e350594/printer/table/printer.go#L43-L45

No ?

Reply:
Indeed, I forgot to remove this one.